### ISSUE TITLE
MusicDatabase query syntax error for artists or albums with quote in name

### ISSUE THREAD

**Issue by Shukuyen** [MPExtended/WifiRemote#3]
Reported by Martin Kucera on [google code](https://code.google.com/p/wifiremote/issues/detail?id=87)
1. Start MediaPortal client
2. Connect to MediaPortal client using aMPdroid
3. Access the Artists view in the Music section of aMPdroid
4. View the albums of an artist which contains quote in name
5. Try playing any of the albums of the selected artist in MediaPortal (PlayOnPc action)

Playback should start, but it doesn't. Error is generated in MP log (see below).
- MediaPortal Version: 1.5.0.0
- WifiRemote Version: 0.7.1.0
- MPExtended Version: 0.5.4
- aMPdroid Version: 1.1.3
- OS: Vista Business x64

Here is the log entry:

```
[2013-10-12 18:02:59,832] [Error ] [12 ] [ERROR] - SQLiteClient: MusicDatabaseV12.db3 cmd:sqlite3_prepare16vm=null err:ERROR detailed:near "s": syntax error query:select * from tracks where strAlbumArtist like '%Blackmore's Night%' AND strAlbum LIKE '%Shadow of the Moon%' order by iTrack ASC
 [2013-10-12 18:02:59,882] [Error ] [12 ] [ERROR] - musicdatabase exception errQLiteClient: MusicDatabaseV12.db3 cmd:sqlite3_prepare16vm=null err:ERROR detailed:near "s": syntax error query:select * from tracks where strAlbumArtist like '%Blackmore's Night%' AND strAlbum LIKE '%Shadow of the Moon%' order by iTrack ASC stack: v SQLite.NET.SQLiteClient.ThrowError(String statement, String sqlQuery, SqliteError err)
 v SQLite.NET.SQLiteClient.Execute(String query)
 v MediaPortal.Music.Database.MusicDatabase.DirectExecute(String aSQL)
 v MediaPortal.Music.Database.MusicDatabase.GetSongsByFilter(String aSQL, List`1& aSongs, String filter)
```

The same issue was already solved in MP core in past in this thread:
http://forum.team-mediaportal.com/threads/apostrophe-in-artist-name.33306/#post-220082

**Comment by Shukuyen** [comment]
Closed with 433f2bf87fc7b4263a010afeaf1baa3e014bf0cc

**Comment by oxan** [comment]
FYI: If you add "closes #3" to a commit message, GitHub automatically closes the issue when you push the commit ;)

**Comment by Shukuyen** [comment]
I know, just forgot about it when comitting. But thanks, you never stop learning :)
